### ISSUE TITLE
Fully Support Parsing of POV Demos

### ISSUE THREAD

**Issue by markus-wa** [markus-wa/demoinfocs-golang#2]
When adding a POV demo to the test demo-set in test/cs-demos the parser crashes.

```
Parsing header
Registering handlers
Parsing to end
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x18 pc=0x744af6]

goroutine 19 [running]:
github.com/markus-wa/demoinfocs-golang/st.(*Entity).ApplyUpdate(0x0, 0xc0428a8000)
        C:/Dev/GoPath/src/github.com/markus-wa/demoinfocs-golang/st/entity.go:53 +0xc6
github.com/markus-wa/demoinfocs-golang.(*Parser).handlePacketEntities(0xc0420ec000, 0xc0425ea0c0)
        C:/Dev/GoPath/src/github.com/markus-wa/demoinfocs-golang/packet_handlers.go:47 +0x2d0
github.com/markus-wa/demoinfocs-golang.(*Parser).(github.com/markus-wa/demoinfocs-golang.handlePacketEntities)-fm(0xc0425ea0c0)
        C:/Dev/GoPath/src/github.com/markus-wa/demoinfocs-golang/parser.go:120 +0x3b
reflect.Value.call(0x79e7e0, 0xc042041040, 0x13, 0x829e20, 0x4, 0xc042c51f60, 0x1, 0x1, 0xc042c51e78, 0xc042c51e70, ...)
        C:/Go/src/reflect/value.go:434 +0x926
reflect.Value.Call(0x79e7e0, 0xc042041040, 0x13, 0xc042c51f60, 0x1, 0x1, 0x4, 0xc04213e000, 0xa53e00)
        C:/Go/src/reflect/value.go:302 +0xab
github.com/markus-wa/godispatch.(*Dispatcher).Dispatch.func1(0xc0420ec068, 0xc042c51f48, 0xc042c51f60, 0x1, 0x1)
        C:/Dev/GoPath/src/github.com/markus-wa/godispatch/dispatch.go:49 +0xa1
github.com/markus-wa/godispatch.(*Dispatcher).Dispatch(0xc0420ec068, 0x811500, 0xc0425ea0c0)
        C:/Dev/GoPath/src/github.com/markus-wa/godispatch/dispatch.go:50 +0x220
github.com/markus-wa/godispatch.(*Dispatcher).dispatchQueue(0xc0420ec068, 0xc04203eb40)
        C:/Dev/GoPath/src/github.com/markus-wa/godispatch/dispatch.go:93 +0x11e
created by github.com/markus-wa/godispatch.(*Dispatcher).AddQueues
        C:/Dev/GoPath/src/github.com/markus-wa/godispatch/dispatch.go:79 +0x189
exit status 2
FAIL    github.com/markus-wa/demoinfocs-golang  0.245s
```

**Comment by markus-wa** [comment]
See `test/cs-demos/broken/POV-orbit-skytten-vs-cloud9-gfinity15sm1-nuke.dem`

**Comment by markus-wa** [comment]
Valve's parser doesn't work for the skytten demo either and I can't even find any other POVs since HLTV removed them from their site last year.

Wont fix (unless someone can provide a demo, maybe).

**Comment by markus-wa** [comment]
Note to self: The Valve parser crashes some time after `LeChris` killing `The Goose` with a CZ-75 while the game actually goes on a fair bit longer.

**Comment by markus-wa** [comment]
@huertasdanny - POV demos are now partially supported.

According to my testing, support is as good or better than other parsers, but still very bad.
Many demos result in a out-of-bounds panic etc. even if they play nicely inside CSGO.

What worked for me:
- local game against bot

What didn't work:
- deathmatch
- casual

Untested:
- wing man
- [dangerzone](https://www.youtube.com/watch?v=siwpn14IE7E)
- war games

**Comment by markus-wa** [comment]
see #227

**Comment by markus-wa** [comment]
Released in v2.3.0 - https://github.com/markus-wa/demoinfocs-golang/releases/tag/v2.3.0
